src/jit/lsraarm.cpp

```diff
@@ -801,6 +828,7 @@ void LinearScan::TreeNodeInfoInit(GenTree* tree)
             info->srcCount         = 2;
             info->dstCount         = 1;
             info->internalIntCount = 1;
+            appendBinaryLocationInfoToList(tree->AsOp());
```

Comment:
Could you please add a header to `appendBinaryLocationInfoToList`? 
Why do we sometimes have `info->srcCount = appendBinaryLocationInfoToList(tree->AsOp());` and sometimes              `info->srcCount         = 2;
appendBinaryLocationInfoToList(tree->AsOp());`?

Reply:
I'll add a header.
Let me look at the cases where we have `info->srcCount = 2; appendBinaryLocationInfoToList(tree->AsOp());`. It may be that those should be changed.